src/App.js

```diff
@@ -1,7 +1,33 @@
+import React, { Component } from "react";
+import ProductDisplay from "./components/ProductDisplay";
+import Cart from "./components/Cart";
+import Checkout from "./components/Checkout";
+import formatPrice from "./helpers/formatPrice"
 import "./App.css";
 
-const App = () => {
-  return <h1>Hello, world!</h1>;
-};
+export default class App extends Component {
+  state = { cart: []}
```

Comment:
I don't remember covering this constructor-less state declaration but cool that you figured it out!